### DISCUSSION TITLE
Error in InstallData("ifnb") - Integration tutorial

### DISCUSSION THREAD

**Discussion by grna92** [satijalab/seurat#7744] [Q&A]
Hi all,

When I try to install the "ifnb" dataset for the scRNAseq integration tutorial I get the following error message:

> InstallData("ifnb")
trying URL 'http://seurat.nygenome.org/src/contrib/ifnb.SeuratData_3.1.0.tar.gz'
Content type 'application/octet-stream' length 413266233 bytes (394.1 MB)
=============================
downloaded 234.4 MB

Error in download.file(url, destfile, method, mode = "wb", ...) : 
  download from 'http://seurat.nygenome.org/src/contrib/ifnb.SeuratData_3.1.0.tar.gz' failed
In addition: Warning messages:
1: In download.file(url, destfile, method, mode = "wb", ...) :
  downloaded length 0 != reported length 0
2: In download.file(url, destfile, method, mode = "wb", ...) :
  URL 'https://seurat.nygenome.org/src/contrib/ifnb.SeuratData_3.1.0.tar.gz': Timeout of 60 seconds was reached
Warning in download.packages(pkgs, destdir = tmpd, available = available,  :
  download of package ‘ifnb.SeuratData’ failed
Error in loadNamespace(name) : 
  there is no package called ‘ifnb.SeuratData’

I have tried rerunning the code but it always gets stuck at the same spot. 

Any workarounds?

Thank you in advance

**Comment by Mosharghi** [comment]
Just increase the timeout with this command:
# Set timeout to 1000 seconds (roughly 16 minutes)
options(timeout = 1000)
Then try to reinstall:
InstallData("ifnb")
I hope it helps.
